src/mame/drivers/prestige.cpp

```diff
@@ -917,7 +928,10 @@ ROM_START( gmmc )
 	ROM_CONTINUE( 0x000000, 0x020000 )
 ROM_END
 
-
+ROM_START( gl1500sl )
+	ROM_REGION(0x100000, "maincpu", 0)
+	ROM_LOAD( "27-5894-01",   0x000000, 0x080000, CRC(7336231c) SHA1(35a1f739994b5c8fb67a7f76d423e50d8154e9ea) )
+ROM_END
```

Comment:
I'm confused, too,  Your added bios rom for gl1500sl is 100% the same as the current gl6000sl in source.
Can you explain how or what brings you to this conclusion that these are the same?